src/lib/ElectrumClient.js

```diff
@@ -451,10 +452,18 @@ export default class Client {
     const transactions = await Promise.all(
       history
         .map(confirmedTx => confirmedTx.tx_hash)
-        .map(txHash => this.getTransaction(txHash))
+        // Catch error so it can proceed to other transactions from the list.
+        // This will produce a `undefined` entry in the list that we need to filter
+        // out.
+        .map(txHash => this.getTransaction(txHash).catch(console.error))
     )
 
-    return transactions
+    // Filter out entries for which `getTransaction` failed in the previous step.
+    /** @type {TransactionData[]} */
+    // @ts-ignore We filtered out void entries.
+    const filteredTransactions = transactions.filter(tx => tx)
```

Comment:
Feels like we could be a little clearer by using `allSettled` and then filtering + logging failed promises, but that's a nitpick.